platform-sdk/swirlds-common/src/main/java/com/swirlds/common/config/EventConfig.java

```diff
@@ -72,4 +72,4 @@ public record EventConfig(
         @ConfigProperty(defaultValue = "/opt/hgcapp/eventsStreams") String eventsLogDir,
         @ConfigProperty(defaultValue = "true") boolean enableEventStreaming,
         @ConfigProperty(defaultValue = "8") int prehandlePoolSize,
-        @ConfigProperty(defaultValue = "true") boolean useLegacyIntake) {}
+        @ConfigProperty(defaultValue = "false") boolean useLegacyIntake) {} // TODO revert (?)
```

Comment:
Note: I've got the new pipeline enabled for testing. Will revert prior to merge.